### ISSUE TITLE
UI Navigation/Accessibility

### ISSUE THREAD

**Issue by terrywbrady** [CDLUC3/mrt-doc#449]
Relates to #448 

Status | Summary | Status
 -- | -- | --
No skip to main content link. | Add one (https://webaim.org/techniques/skipnav/).  | Deployed to stage 8/31
Global nav droplists can't be opened via keyboard. | Change nav droplists to open on click/return and use tab/shift+tab to navigate, or to open on spacebar and use up/down arrow keys to navigate. (https://webaim.org/techniques/keyboard/)  | Deployed to stage 9/1.  What is the proper way to tell the user to use spacebar?
"Logged in as Guest" droplist in global nav skipped by keyboard tab-navigation. This makes logging-out impossible. | Users should be able use tab to put focus on the "Logged in as Guest" menu item. | Deployed to stage 9/1
Open/close menu control ("hamburger" icon) skipped by keyboard (when browser is in narrow screen/mobile mode). This renders the global nav completely unusable under some circumstances. | Users should be able use tab to put focus on the open/close menu control. | Deployed to stage 9/1.  What is the proper way to tell the user to use spacebar?

**Comment by elopatin-uc3** [comment]
@terrywbrady Rachael and I reviewed the first four items that are on stage. The upshot is that these all look good. With regard to your question about the appropriate way to let users know to use the spacebar, Rachael's feedback was that the presence of the downward facing rectangle should be enough to indicate to a user that a secondary action (such as the spacebar) is needed. Also, in general, working with the expectation that a user will attempt use of the spacebar should be fine. That said, she will go back over this, specifically with a screen reader to provide any other feedback.

**Comment by elopatin-uc3** [comment]
Another note (which does not need to be addressed for this release): The current order in which a user's tab focus moves is from the Merritt icon, to Logged in as.., to Collection home, etc. It may be more appropriate to move from the Merritt icon, to Collection home, and continue in a left to right fashion from there.
